app/models/category_list.rb

```diff
@@ -70,7 +70,7 @@ def self.order_categories(categories)
 
   private
 
-  def find_relevant_topics
+  def all_relevant_topics
```

Comment:
This wording makes me think that `all_relevant_topics` is going to return all and `find_relevant_topics` is a way to filter down from there. I wonder if renaming `all_relevant_topics` -> `relevant_topics_query` or something would be more clear?

Reply:
Good call. Naming things is hard. 😄